src/Mvc/build/dependencies.props

```diff
@@ -10,91 +10,91 @@
   
     <AngleSharpPackageVersion>0.9.9</AngleSharpPackageVersion>
     <BenchmarkDotNetPackageVersion>0.10.13</BenchmarkDotNetPackageVersion>
-    <BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>3.0.0-preview.18572.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>
-    <BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlitePackageVersion>3.0.0-preview.18572.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlitePackageVersion>
-    <BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlServerPackageVersion>3.0.0-preview.18572.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreSqlServerPackageVersion>
-    <BenchmarksOnlyMySqlConnectorPackageVersion>0.47.1</BenchmarksOnlyMySqlConnectorPackageVersion>
+    <BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>2.1.1</BenchmarksOnlyMicrosoftEntityFrameworkCoreDesignPackageVersion>
```

Comment:
This seems wrong. Going from 3.0.0 - 2.1.1? @sebastienros  expected?

Reply:
We should not change these `BenchmarksOnly` versions. Unless it causes an issue?

Reply:
These values were automatically updated by running `./run.cmd -Path src/Mvc/ -LockFile ./korebuild-lock.txt upgrade deps`

@natemcmaster @ryanbrandenburg @sebastienros Is the upgrade deps tool putting the wrong values here?

Reply:
I will look at the script, maybe that's what I should have changed, didn't know it was updating these also. Just revert these lines in the meantime.

Reply:
Done